docs/src/main/paradox/release-notes/releases-1.1.md

```diff
@@ -5,6 +5,11 @@
 Release notes for Apache Pekko 1.1.0-M1. See [GitHub Milestone](https://github.com/apache/pekko/milestone/2?closed=1) for fuller list of changes.
 As with all milestone releases, this release is not recommended for production use - it is designed to allow users to try out the changes in a test environment.
 
+### Additional APIs
```

Comment:
why separate these from the other 'Additional APIs' below?

Reply:
I thought thats what you were asking for, I will move it back

Reply:
Resolved and pushed.